fsw/public_inc/crypto_error.h

```diff
@@ -0,0 +1,23 @@
+/* Copyright (C) 2009 - 2017 National Aeronautics and Space Administration. All Foreign Rights are Reserved to the U.S. Government.

```

Comment:
Note for us - we need to update copyrights and change the email address to the new mailing list.